src/shared/components/barGraph/BarGraph.tsx

```diff
@@ -170,7 +171,7 @@ export default class BarGraph extends React.Component<IBarGraphProps, {colors: s
                     cursor: 'pointer',
                     events: {
                         click: function() {
-                            location.href= 'http://www.cbioportal.org/study?id=' + this.options.studyId, + '#summary';
+                            window.location.href= 'http://www.cbioportal.org/study?id=' + this.options.studyId, + '#summary';
```

Comment:
just fyi, this should be relative or configurable so we can point it a particular instance.  relative is probably the best way to go.